plugin.py

```diff
@@ -54,3 +76,18 @@ def find_package_dependency_dirs() -> List[str]:
         dep_dirs.append(packages_path)
 
         return [path for path in dep_dirs if os.path.isdir(path)]
+
+    def _start_indicator(self, msg: str = "") -> None:
+        if self._activity_indicator:
+            self._activity_indicator.label = msg  # type: ignore
```

Comment:
Interesting that it doesn't have a public API for updating the label. But then `label`, even though not documented, is likely public as it doesn't have an underscore prefix.